PowerUp/app/src/main/res/layout/gamemap.xml

```diff
@@ -39,21 +39,22 @@
         android:id="@+id/BoyfriendButton"
         android:layout_width="@dimen/map_click_width"
         android:layout_height="@dimen/map_click_height"
-        android:layout_above="@id/ContinueButtonMapGameBegins"
-        android:layout_alignLeft="@id/HouseButton"
-        android:layout_alignStart="@id/HouseButton"
+
         android:alpha="0.00"
-        android:text="@string/boyfriend" />
+        android:text="@string/boyfriend"
+        android:layout_alignTop="@+id/SchoolButton"
+        android:layout_alignLeft="@+id/HouseButton"
+        android:layout_alignStart="@+id/HouseButton" />
 
     <Button
         android:id="@+id/SchoolButton"
         android:layout_width="@dimen/map_click_width"
         android:layout_height="@dimen/map_click_height"
-        android:layout_alignBottom="@id/BoyfriendButton"
-        android:layout_alignRight="@id/ContinueButtonMapGameBegins"
-        android:layout_alignParentRight="true"
-        android:layout_marginBottom="@dimen/school_button_margin_bottom"
+        android:layout_marginBottom="82dp"
```

Comment:
Don't use raw values here. If you want to add a margin, you should instead declare the value in dimens.xml and then refer it from here as with the width and height.
If you are using the layout editor, I recommend you not to use it because it sometimes introduces raw values for you.